packages/plugin-user/src/config.js

```diff
@@ -5,14 +5,19 @@
  */
 
 export default {
+  device: {
+    preDiscoveryServices: {
+      atlasServiceUrl: process.env.ATLAS_SERVICE_URL || `https://atlas-a.wbx2.com/admin/api/v1`
+    }
+  },
   user: {
+    activationUrl: `https://idbroker.webex.com/idb/token/v1/actions/UserActivation/invoke`,
     batcherWait: 100,
     batcherMaxCalls: 100,
     batcherMaxWait: 1500,
+    setPasswordUrl: `https://identity.webex.com/identity/scim/v1/Users`,
     verifyDefaults: {
-      deviceName: `DESKTOP`,
-      deviceId: `not a mobile device`,
-      pushId: `not a mobile device`
+      reqId: `DESKTOP`
```

Comment:
reqId is required in the atlas endpoint. it is supposed to be used to check if the email has been delivered. https://wiki.cisco.com/display/WX2/New+Onboarding+Flows
but it is currently not used in the web client